tests/parsers/moldev_softmax_pro/testdata/MD_SMP_absorbance_endpoint_example01.json

```diff
@@ -30,7 +30,7 @@
                             }
                         }
                     ],
-                    "measurement time": "1970-01-01T00:00:00-00:00",
+                    "measurement time": "1970-01-01T00:00:00+00:00",
```

Comment:
Note to reviewer: The change in the `measurement time` on all files is due to the fact that the softmaxpro did not use the `timestamp_parser` but instead passed the `EPOCH` for all measurements, this changed in the schema mapper.

 The default `teimestamp_parser` uses `dateutil.parser` to parse the dates, which prefers the `+00:00` offset instead of `-00:00`